app/components/curriculum-inventory-verification-preview.hbs

```diff
@@ -1,111 +1,128 @@
+        <span class="link clickable" role="button" {{on "click" (fn this.scrollTo "table3b")}}>
           {{t "general.table3bClerkshipSequenceBlockInstructionalTime"}}
-        </a>
+        </span>
       </li>
       <li>
-        <a href="#verification-preview-table4">
+        <span class="link clickable" role="button" {{on "click" (fn this.scrollTo "table4")}}>
           {{t "general.table4InstructionalMethodCounts"}}
-        </a>
+        </span>
       </li>
       <li>
-        <a href="#verification-preview-table5">
+        <span class="link clickable" role="button" {{on "click" (fn this.scrollTo "table5")}}>
           {{t "general.table5NonClerkshipSequenceBlockAssessmentMethods"}}
-        </a>
+        </span>
       </li>
       <li>
-        <a href="#verification-preview-table6">
+        <span class="link clickable" role="button" {{on "click" (fn this.scrollTo "table6")}}>
           {{t "general.table6ClerkshipSequenceBlockAssessmentMethods"}}
-        </a>
+        </span>
       </li>
       <li>
-        <a href="#verification-preview-table7">
+        <span class="link clickable" role="button" {{on "click" (fn this.scrollTo "table7")}}>
           {{t "general.table7AllEventsWithAssessmentsTaggedAsFormativeOrSummative"
           }}
-        </a>
+        </span>
       </li>
       <li>
-        <a href="#verification-preview-table8">
+        <span class="link clickable" role="button" {{on "click" (fn this.scrollTo "table8")}}>
           {{t "general.table8AllResourceTypes"}}
-        </a>
+        </span>
       </li>
     </ul>
+    <span class="hidden" {{did-insert this.setScrollTarget "table1"}}></span>
     <CurriculumInventoryVerificationPreviewTable1
       @data={{this.tables.program_expectations_mapped_to_pcrs}}
     />
-    <a class="back-to-toc" href="#{{this.tocId}}">
+    <span class="link clickable back-to-toc" role="button" {{on "click" (fn this.scrollTo "toc")}}>
       {{t "general.backToTableOfContents"}}
-    </a>
+    </span>
+    <span class="hidden" {{did-insert this.setScrollTarget "table2"}}></span>
     <CurriculumInventoryVerificationPreviewTable2
       @data={{this.tables.primary_instructional_methods_by_non_clerkship_sequence_blocks
       }}
     />
-    <a class="back-to-toc" href="#{{this.tocId}}">
+    <span class="link clickable back-to-toc" role="button" {{on "click" (fn this.scrollTo "toc")}}>
       {{t "general.backToTableOfContents"}}
-    </a>
+    </span>
+    <span class="hidden" {{did-insert this.setScrollTarget "table3a"}}></span>
     <CurriculumInventoryVerificationPreviewTable3a
       @data={{this.tables.non_clerkship_sequence_block_instructional_time}}
     />
-    <a class="back-to-toc" href="#{{this.tocId}}">
+    <span class="link clickable back-to-toc" role="button" {{on "click" (fn this.scrollTo "toc")}}>
       {{t "general.backToTableOfContents"}}
-    </a>
+    </span>
+    <span class="hidden" {{did-insert this.setScrollTarget "table3b"}}></span>
```

Comment:
I think (but am not certain) that you could put this `did-insert` onto the component as:
```
<CurriculumInventoryVerificationPreviewTable4
   @data={{this.tables.instructional_method_counts}}
  {{did-insert this.setScrollTarget "table3b"}}
/>
```
as long as the CurriculumInventoryVerificationPreviewTable4 has an `...attributes` it will apply `did-insert` where `...attributes` is in the component.

Reply:
this works. brilliant!